grails-app/controllers/au/org/ala/biocollect/BioActivityController.groovy

```diff
@@ -1879,7 +1882,7 @@ class BioActivityController {
         } else if (projectService.isUserAdminForProject(userId, projectId) || activityService.isUserOwnerForActivity(userId, activity?.activityId)) {
             model = [activity: activity]
             if (includeSiteData) {
-                model = activityAndOutputModel(activity, activity.projectId)
+                model.site = model.activity?.siteId ? siteService.get(model.activity.siteId, [view:'brief']) : null
```

Comment:
siteService.get() returns a lot of properties. Use collect to return only necessary properties.

Reply:
@temi what can i use to get the values for geoJson?
if (includeSiteData) {
                model.site = siteService.get(activity.siteId).collect {
                    [
                            siteId: it.siteId,
                            name : it.name
                    ]
                }
            }

Reply:
geoJSON can be found in property geoIndex.